typings/search.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface Search {
```

Comment:
Any reason you named it `search` and not `queryOptions` for example?

Reply:
No there isn't any reason for it. I'm happy with queryOptions so I changed id into it.